config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  root 'homepages#index'
+  resources :works
+  resources :homepages
+  resources :users
```

Comment:
I don't think you need all 7 RESTful routes for `homepages` and `users` for this app.